src/main/java/core/basesyntax/service/FileReadImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service;
+
+import core.basesyntax.service.impl.FileRead;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReadImpl implements FileRead {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
```

Comment:
Consider using `Files.newBufferedReader(Paths.get(fileName))` instead of `new BufferedReader(new FileReader(fileName))` to take advantage of the newer java.nio package, which provides better performance and error handling.